tests/gcp/hooks/test_bigquery.py

```diff
@@ -80,66 +72,6 @@ def test_credentials_provided(self, mock_read_gbq, _):
         self.assertEqual("PROJECT_ID", kwargs['project_id'])
 
 
-class TestBigQueryDataframeResults(unittest.TestCase):
```

Comment:
Why did you delete it?

Reply:
It is moved to `tests/gcp/hooks/test_bigquery_system.py` due to the fact that they require BigQuery connection. I forgot to add them to commit, I've made fixup.